blocks/article-body-block/chains/article-body/_children/pullquote.jsx

```diff
@@ -11,7 +11,7 @@ export default ({ element }) => {
       pullQuote.push(<p key={paragraph._id}>{ paragraph.content }</p>);
     }
   });
-  if (citation.type === 'text') {
+  if (citation.type === 'text' && citation.content !== (null || undefined || '')) {
```

Comment:
<img width="322" alt="Screen Shot 2021-04-28 at 13 44 35" src="https://user-images.githubusercontent.com/5950956/116456379-d5421200-a827-11eb-9d59-9ed0309f0b96.png">

that `null || undefined || ''` will evaluate as "" because null and undefined are both falsy

<img width="292" alt="Screen Shot 2021-04-28 at 13 45 10" src="https://user-images.githubusercontent.com/5950956/116456449-ebe86900-a827-11eb-9414-7add54918520.png">

Reply:
I always look at this chart to doublecheck falsy values https://dorey.github.io/JavaScript-Equality-Table/